gps/metrics.go

```diff
@@ -31,15 +31,15 @@ func newMetrics() *metrics {
 
 func (m *metrics) push(name string) {
 	cn := m.stack[len(m.stack)-1]
-	m.times[cn] = m.times[cn] + time.Since(m.last)
+	m.times[cn] += +time.Since(m.last)
```

Comment:
What's the purpose of the `+` before the `time` here?

Reply:
Oh, sorry. It should be omitted. Thanks for spotting.